### PR TITLE
more models; spell check

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: tests/testthat/test-get_data.R [hunk 1]
```diff
@@ -2,8 +2,9 @@ if (require("testthat") && require("insight") && require("lme4")) {
   data("cbpp")
   set.seed(123)
   cbpp$cont <- rnorm(nrow(cbpp))
-  m <- glmer(cbind(incidence, size - incidence) ~ poly(cont,2) + (1 | herd),
-             data = cbpp, family = binomial)
+  m <- glmer(cbind(incidence, size - incidence) ~ poly(cont, 2) + (1 | herd),
+    data = cbpp, family = binomial
+  )
```
Comment (strengejacke) [inline tests/testthat/test-get_data.R:8]: That's a strange rule.

Comment (strengejacke) [reply tests/testthat/test-get_data.R:8]: I suggest keeping the previous version.

Comment (IndrajeetPatil) [reply tests/testthat/test-get_data.R:8]: Hmm, good catch. This might actually be a bug in `styler`.

Comment (IndrajeetPatil) [reply tests/testthat/test-get_data.R:8]: Apparently, not. This is what the `tidyverse` style guide dictates:
https://github.com/tidyverse/style/issues/39

Do you still want me to revert this change?

Comment (strengejacke) [reply tests/testthat/test-get_data.R:8]: In the end, it's just a test, but I wouldn't format my other code that way... And the rule you linked to is not "working" here, as `family` is also a named argument, but in the same line.

**Pull request merged**